README.md

```diff
@@ -7,49 +7,27 @@ utilize React.
 
 
 ## Build a new application using the LTI Starter App Rails application template:
-Since the project is not public you will need to have a local copy of the
-lti_starter_app project. Be sure to use the correct relative path:
-
-`rails new my_app -m ./lti_starter_app/template.rb`
-
+-----------
+`rails new my_app -m https://raw.githubusercontent.com/atomicjolt/lti_starter_app/master/template.rb`
 
 ## Running LTI Starter App
 -----------
-
-### With Foreman
-Foreman makes it simple to startup all the services required to run the
-application in development mode. To start the application using foreman simply
-run:
+See [Setting up React Rails Starter App](#setup) below to configuration the application.
+Once setup Start Rails and the webpack server:
 
 ```
-$ foreman start -f Procfile.dev
+  rails server
+  yarn hot
 ```
 
-Make sure you have the latest version of Foreman installed.
-
-#### Environment
-Foreman will automatically find and read the .env file.
-
-### Without Foreman
-If you need to run services individually or just don't like Foreman you can run
-each service seperately:
-
-```
-$ rails server
-$ npm run hot
-```
-
-## Setting up LTI Starter App
+## <a name="setup"></a>Setting up LTI Starter App
 -----------
 
 ### File Modifications
+-----------
 
-#### Change bin/bootstrap
```

Comment:
Let's add this back in. It's only relevant for Atomic Jolt team members, but it's important information.